### DISCUSSION TITLE
Telescope terms behavior

### DISCUSSION THREAD

**Discussion by georgejean** [NvChad/NvChad#2076] [Ideas]
I have experimented terminal toggling with `<M-i>`, `<M-v>`, `<M-h>` and `<leader>pt`. It's great !
But I think it could be natural to enable terminals selection via Telescope in the case terms are already opened.
What do you think about it?

**Comment by siduck** [comment]
@georgejean terminal selection i.e leader pt will only show hidden terminals

**Comment by georgejean** [comment]
I know but it was just a suggestion to extend its capability (enable focus on an opened term when using `<leader>pt`).
It's very good like that anyway !
